Formula/prime_server.rb

```diff
@@ -0,0 +1,26 @@
+class PrimeServer < Formula
+  desc "non-blocking (web)server API for distributed computing and SOA based on zeromq"
+  homepage "https://github.com/kevinkreiser/prime_server"
+  head "https://github.com/kevinkreiser/prime_server.git", tag: "0.6.3"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "gcc" => :build
+  depends_on "pkg-config" => :build
+  depends_on "curl"
+  depends_on "czmq"
+
+  def install
+    system "./autogen.sh"
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    pipe_output("prime_serverd").include?("Usage: prime_serverd num_requests|server_listen_endpoint concurrency")
```

Comment:
New formulae in homebrew-core are required to have a functionality test, not just a `help`, `version`, or `usage` test. Check out [zeromq's test block](https://github.com/Homebrew/homebrew-core/blob/30abddeed8f2b8d6e3d7b5ee313ddcd2c8ff1671/Formula/zeromq.rb#L55) as an example of this.